flutter/shell/platform/tizen/tizen_renderer.h

```diff
@@ -26,8 +26,6 @@ class TizenRenderer {
 
   bool IsValid() { return is_valid_; }
 
-  FlutterDesktopRendererType type() { return type_; }
-
```

Comment:
You can remove this unnecessary include:
```cpp
#include "flutter/shell/platform/tizen/public/flutter_tizen.h"
```

Reply:
done